pystarport/pystarport/convert.sh

```diff
@@ -1,17 +1,13 @@
 #!/bin/bash
 OUTPUT=./proto_python
-COSMOS=./cosmos-sdk
+COSMOS=../../third_party/cosmos-sdk
 TENDERMINT=./tendermint
 TMP=$(whereis grpc_python_plugin)
 PLUGIN="$(cut -d' ' -f2 <<<"$TMP")"
 mkdir $OUTPUT
-git clone --branch v0.41.0 https://github.com/cosmos/cosmos-sdk.git
 git clone --branch v0.34.3 https://github.com/tendermint/tendermint.git
-cp -Rf $COSMOS/third_party/proto/* $COSMOS/proto/ 
-rm -rf $COSMOS/proto/cosmos/base/tendermint
 # cosmos
-protoc --proto_path=$COSMOS/proto --proto_path=$COSMOS/third_party/proto --python_out=$OUTPUT $(find $COSMOS/proto/cosmos -iname "*.proto") --grpc_python_out=$OUTPUT  --plugin=protoc-gen-grpc_python=$PLUGIN
+python -m grpc.tools.protoc --proto_path=$COSMOS/proto --proto_path=$COSMOS/third_party/proto --python_out=$OUTPUT $(find $COSMOS/proto/cosmos -iname "*.proto") --grpc_python_out=$OUTPUT  --plugin=protoc-gen-grpc_python=$PLUGIN
```

Comment:
Do we need to install some python library to run this? If yes, maybe edit readme with required dependencies.

Reply:
this script is normally run by `nix-shell` automatically, the environment is setup in [here](https://github.com/crypto-org-chain/chain-main/pull/384/files#diff-5819221f7ad475c47c88c3293eb44a46586748a892d982474398550247f63b13R4)